src/components/pages/donation_confirmation/AddressUpdateForm.vue

```diff
@@ -210,7 +210,10 @@ const { addressType, addressTypeIsInvalid, setAddressType } = useAddressTypeFunc
 const addressTypeModel = ref<AddressTypeModel>( addressType.value );
 watch( addressTypeModel, ( newAddressType: AddressTypeModel ) => setAddressType( newAddressType ) );
 
-const mailingList = useMailingListModel( store );
+// const mailingList = useMailingListModel( store );
+
+// TODO: get default value from somewhere (backend or frontend)
```

Comment:
You can remove these comments

Reply:
there is still a TODO left, so there is a default value in the code now, is this still a TODO to get it from somewhere else?
if yes, maybe that could be specified deeper in the comment